docs/root/configuration/http_filters/grpc_json_transcoder_filter.rst

```diff
@@ -144,6 +144,6 @@ gRPC or RESTful JSON requests to localhost:51051.
           - endpoint:
               address:
                 socket_address:
-                  address: docker.for.mac.localhost
+                  address: host.docker.internal
```

Comment:
Could you also add a `warning` message for users using older versions of Docker to use `docker.for.mac.localhost` instead? It'd also be good to point users to a relevant document on Docker's website.